### PR TITLE
Read Staking, Token address from database

### PR DESCRIPTION
**Pull request by rajeevbbqq**

### CODE REVIEW

File: airdrop/config.py [hunk 1]
```diff
@@ -24,8 +24,6 @@
 SIGNER_PRIVATE_KEY = 'AIRDROP_SIGNER_PRIVATE_KEY'
 SIGNER_PRIVATE_KEY_STORAGE_REGION = ''
 
-NUNET_TOKEN_ADDRESS = ''
-STAKING_CONTRACT_ADDRESS = ''
 STAKING_TOKEN_NAME = 'AGIX'
```
Comment (prashantramangupta) [inline airdrop/config.py:27]: STAKING_TOKEN_NAME
Should come from the database.

File: airdrop/infrastructure/repositories/airdrop_repository.py [hunk 2]
```diff
@@ -165,7 +165,15 @@ def get_token_address(self, airdrop_id):
         if airdrop is None:
             raise Exception("Airdrop not found")
 
-        return airdrop.address
+        return airdrop.token_address
+
+    def get_staking_contract_address(self, airdrop_id):
```
Comment (prashantramangupta) [inline airdrop/infrastructure/repositories/airdrop_repository.py:170]: This is not the right way to query the database.

File: airdrop/config.py (hunk 1 above)
Comment (rajeevbbqq) [reply airdrop/config.py]: Moved to db in the latest commit

File: airdrop/infrastructure/repositories/airdrop_repository.py (hunk 2 above)
Comment (rajeevbbqq) [reply airdrop/infrastructure/repositories/airdrop_repository.py:182]: Fixed in the latest commit

**Review by prashantramangupta — APPROVED** [review]
Approved

**Pull request merged**